templates/beta/contracts/BetaTemplateBase.sol

```diff
@@ -42,43 +52,118 @@ contract BetaTemplateBase {
+        InstalledApp(tokenManager, appIds[uint8(Apps.TokenManager)]);
 
         token.changeController(tokenManager); // sender has to create tokens
 
         // permissions
-        acl.createPermission(ANY_ENTITY, voting, voting.CREATE_VOTES_ROLE(), voting);
-        acl.createPermission(voting, voting, voting.MODIFY_QUORUM_ROLE(), voting);
-
-        acl.createPermission(finance, vault, vault.TRANSFER_ROLE(), voting);
-        acl.createPermission(voting, finance, finance.CREATE_PAYMENTS_ROLE(), voting);
-        acl.createPermission(voting, finance, finance.EXECUTE_PAYMENTS_ROLE(), voting);
-        acl.createPermission(voting, finance, finance.DISABLE_PAYMENTS_ROLE(), voting);
-        acl.createPermission(voting, tokenManager, tokenManager.ASSIGN_ROLE(), voting);
-        acl.createPermission(voting, tokenManager, tokenManager.REVOKE_VESTINGS_ROLE(), voting);
+        acl.createPermission(
+            ANY_ENTITY,
+            voting,
+            voting.CREATE_VOTES_ROLE(),
+            voting
+        );
+        acl.createPermission(
+            voting,
+            voting,
+            voting.MODIFY_QUORUM_ROLE(),
+            voting
+        );
+
+        acl.createPermission(
+            finance,
+            vault,
+            vault.TRANSFER_ROLE(),
+            voting
+        );
+        acl.createPermission(
+            voting,
+            finance,
+            finance.CREATE_PAYMENTS_ROLE(),
+            voting
+        );
+        acl.createPermission(
+            voting,
+            finance,
+            finance.EXECUTE_PAYMENTS_ROLE(),
+            voting
+        );
+        acl.createPermission(
+            voting,
+            finance,
+            finance.DISABLE_PAYMENTS_ROLE(),
+            voting
+        );
+        acl.createPermission(
+            voting,
+            tokenManager,
+            tokenManager.ASSIGN_ROLE(),
+            voting
+        );
+        acl.createPermission(
+            voting,
+            tokenManager,
+            tokenManager.REVOKE_VESTINGS_ROLE(),
+            voting
+        );
 
         require(holders.length == stakes.length);
 
-        acl.createPermission(this, tokenManager, tokenManager.MINT_ROLE(), this);
+        acl.createPermission(
+            this,
+            tokenManager,
+            tokenManager.MINT_ROLE(),
+            this
+        );
 
-        tokenManager.initialize(token, _maxTokens > 1, _maxTokens, true);
+        tokenManager.initialize(
+            token,
+            _maxTokens > 1,
+            _maxTokens,
+            true
+        );
 
         for (uint256 i = 0; i < holders.length; i++) {
             tokenManager.mint(holders[i], stakes[i]);
         }
 
         // inits
-        finance.initialize(vault, etherToken, uint64(-1)); // yuge period
+        finance.initialize(vault, etherToken, uint64(-1) - uint64(now)); // yuge period
```

Comment:
👍Awesome! Using safemath in the Finance app also makes a lot of sense to catch stuff like this.